pkg/engine/tunnel.go

```diff
@@ -104,6 +107,24 @@ func (t *TunnelEngine) clearDriver() error {
 	return nil
 }
 
+func (t *TunnelEngine) getStunInfo() error {
```

Comment:
How about adjust `getStunInfo` to  `checkNatCapability`？

Reply:
Sounds good.